src/exchange/resource/resource.service.js

```diff
@@ -1,10 +1,9 @@
 angular.module('Module.exchange.services').service(
   'ExchangeResources',
   class ExchangeResources {
-    constructor($rootScope, Products, $http, $q, Exchange, OvhHttp) {
+    constructor($rootScope, $http, $q, Exchange, OvhHttp) {
```

Comment:
As they are not too many injections, maybe they could have been sorted ?

Reply:
Done